sim/Makefile

```diff
@@ -35,20 +41,31 @@ $(FIRRTL_JAR): $(shell find $(FIRRTL_SUBMODULE_DIR)/src/main/scala -iname "*.sca
 firrtl: $(FIRRTL_JAR)
 .PHONY: firrtl
 
-# Phony targets for launching the sbt shell and running scalatests
-sbt: $(FIRRTL_JAR)
-	$(SBT) $(SBT_FLAGS) shell
-test: $(FIRRTL_JAR)
-	$(SBT) $(SBT_FLAGS) test
+else
 
-PLATFORM := f1
+$(info FireSim as a library)
+
+# REBAR make variables
+base_dir := $(firesim_base_dir)/../../
+sim_dir := $(firesim_base_dir)
+include $(base_dir)/Makefrag
+
+endif
 
 # Include target-specific sources and input generation recipes
 include $(TARGET_PROJECT_MAKEFRAG)
 
 verilog: $(VERILOG)
 compile: $(VERILOG)
 
+# Phony targets for launching the sbt shell and running scalatests
+sbt: $(FIRRTL_JAR)
```

Comment:
Do you ever mark these as phony with `.PHONY`?

Reply:
Not this Makefile, will fix.